lib/shard/src/proxy_segment/segment_entry.rs

```diff
@@ -948,60 +948,6 @@ impl SegmentEntry for ProxySegment {
         self.write_segment.get().read().check_error()
     }
 
-    fn delete_filtered<'a>(
```

Comment:
Self-check: can we actually remove this?

Reply:
Yes, instead we select point IDs and delete those points directly:

https://github.com/qdrant/qdrant/blob/5936926c885cdfc6446d158eac50c2befa508026/lib/collection/src/collection_manager/segments_updater.rs#L785